expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java

```diff
@@ -113,4 +114,8 @@ private Stream<String> fetchContentRecursively(URI uri) {
     private Optional<String> fetch(URI externalReference) {
         return uriRetriever.getRawContent(externalReference, APPLICATION_JSON_LD.toString());
     }
+
+    private Optional<String> fetchOrganizations(URI externalReference) {
+        return uriRetriever.getRawContent(externalReference, APPLICATION_JSON_LD.toString() + CRISTIN_VERSION);
+    }
```

Comment:
Not sure what we are expecting here, as I can't find a single mock response that tells me what is expected from this.

Maybe I am just not finding the mock responses?